### PR TITLE
add helm ct install. 

### PR DESCRIPTION
**Pull request by knelasevero**
First step, see https://github.com/kubernetes-sigs/descheduler/pull/895#issuecomment-1203608848

Ater this PR is merged we can check it prow considers gh-actions when deciding to merge. After that we can go forward with the other changes from https://github.com/kubernetes-sigs/descheduler/pull/895

### CODE REVIEW

**Comment by knelasevero** [comment]
It wont show meaningful info in this CI since https://github.com/kubernetes-sigs/descheduler/pull/895/files#r934597809 was removed and we don't have chart changes.

File: .github/workflows/helm.yaml [hunk 1]
```diff
@@ -39,6 +43,22 @@ jobs:
...
           fi
-
+      
       - name: Run chart-testing (lint)
         run: ct lint --config=.github/ci/ct.yaml --validate-maintainers=false
+      
+      # Need a multi node cluster so descheduler runs until evictions
+      - name: Create multi node Kind cluster
+        run: |
+          kind create cluster --name kind --config ./hack/kind_config.yaml --wait 2m
```
Comment (a7i) [inline .github/workflows/helm.yaml:53]: @knelasevero what step install the `kind` cli? and how can I control the verison?

Comment (knelasevero) [reply .github/workflows/helm.yaml:53]: Not sure if kind comes pre-baked in this gh-actions image or if it is installed by one of the previous actions :thinking:

**Pull request merged**